### DISCUSSION TITLE
¿Does Parcel recognizes and bundles Blade templates from Laravel?

### DISCUSSION THREAD

**Discussion by Rulios** [parcel-bundler/parcel#6022] [Q&A]
Hi

Right now I'm setting all up for a project that uses Laravel PHP and JS. I really like Parcel as a bundler since all other ones are very laborious to start everything up. Just wondering if it recognizes embedded script tags in Blade template from Laravel.

Have a good day!

**Comment by devongovett** [comment]
I don't really know anything about Laravel, but my guess is that you'll need to write a plugin for it.
